frontend/src/component/admin/apiToken/ProjectsList/ProjectsList.tsx

```diff
@@ -67,16 +86,40 @@ export const ProjectsList: FC<IProjectsListProps> = ({ projects, project }) => {
         return <LinkCell to={`/projects/${item}`} title={item} />;
     }
 
+    const getTitle = () => {
+        if (!isWildcard && !isLegacy) {
+            return (
+                <>
+                    ALL current and future projects. This is an orphaned token
+                    with it's project deleted. This token gives more access than
+                    it was intended to and should be deleted.
+                </>
+            );
+        }
+
+        if (isLegacy) {
+            return (
+                <>
+                    ALL current and future projects. This token has v1 format.
+                    Read more about{' '}
+                    <a href='https://docs.getunleash.io/reference/api-tokens-and-client-keys#format'>
+                        token formats
+                    </a>
```

Comment:
I'm not sure I understand why we're showing this. Is it important for the user to know? Also, why should they go and read about the formats? The text doesn't seem actionable.